### DISCUSSION TITLE
Recurring events having weird reminders

### DISCUSSION THREAD

**Discussion by jackthebean** [FossifyOrg/General-Discussion#457] [General]
I use a lot of recurring events in my calendar but don't know why, at some point, after creating such an event the time for reminders changes enormously.

![Screenshot_20241202_161104_org fossify calendar](https://github.com/user-attachments/assets/bf099a3a-52ab-43a2-8525-3969a4cc4da6)
![Screenshot_20241202_161203_org fossify calendar](https://github.com/user-attachments/assets/94bf320d-d5de-4811-bd3c-10573623913e)
![Screenshot_20241202_161237_org fossify calendar](https://github.com/user-attachments/assets/9673e731-00d3-4bd9-a15d-16e5e3948be7)

When these events were created the reminders were probably set for 1 hr before and 14 hours before but I cannot be sure as all the events have now changed to the odd negative number shown.
This happens to multiple different events and I have another one which also showed a similar but not quite the same negative reminders.
Does anyone know why and/or how I can stop this and/or whether it is a formal issue requiring a fix/bug issue raising.
Thanks,

**Comment by Aga-C** [comment]
Please try the following steps:
- Export this event to an ICS file.
- Delete event in Calendar.
- Import it.

Did it help?

**Comment by jackthebean** [comment]
Thanks for your help.   Exported by 'Share', 'Save As' and then deleted all occurrences of that event.  Then imported via 'Settings', 'Import events from an .ics file' and event (and all repetitions) was imported.   Event looked identical with the same reminder numbers in all event dates that I checked.

**Reply by Aga-C** [reply]
Please report it as a bug. If possible, please add the following information to your bug report:
- Attach to it this wrong ICS file. 
- If you're able to create new events with the same problem, provide reproduction steps. 
- It may also be helpful for us if we'll know if this event is local or synchronized via CalDAV. And if it is, what app for synchronization are you using.

Thanks!

**Comment by jackthebean** [comment]
Bug report created.  Thanks.
